### ISSUE TITLE
LayoutNode reference inside subclass

### ISSUE THREAD

**Issue by silentninja** [schibsted/layout#79]
Is there a way to access the node inside a custom subclass of a UITableViewCell or a UITableView. I need to subclass a UITableViewCell to add actions and outlets specific for a cell and need access to the node to set/update the node data when an action is triggered. 
 ```
//when accessing layoutNode
self.layoutNode?.setState()

Error:(28, 20) 'layoutNode' is inaccessible due to 'fileprivate' protection level
```

**Comment by nicklockwood** [comment]
I've made the `layoutNode` property public. If you link against the develop branch by putting this in your Podfile, you can use the fixed version:

    pod 'Layout', :git => 'https://github.com/schibsted/Layout.git', :branch => 'develop'

**Comment by nicklockwood** [comment]
Fixed in 0.6.0
